Example/Info.plist

```diff
@@ -30,7 +30,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>42</string>
+	<string>43</string>
```

Comment:
Can you update the version in the Zone5.xcodeproj/project.pbxproj as well?
see here:
https://github.com/Zone5-Cloud/z5-sdk-swift/pull/65/files

Reply:
Done c2ba39e